src/components/PeopleContent.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Loader } from './Loader';
+import { Person } from '../types';
+
+type Props = {
+  people: Person[],
+  isError: boolean,
+  isLoading: boolean,
+};
+
+export const PeopleContent: React.FC<Props> = ({
+  people,
+  isError,
+  isLoading,
+}) => {
+  const { personId } = useParams();
+
+  if (isError) {
+    return (
+      <p data-cy="peopleLoadingError" className="has-text-danger">
+        Something went wrong
+      </p>
+    );
+  }
+
+  if (isLoading) {
+    return (
+      <Loader />
+    );
+  }
+
+  if (!people.length) {
+    return (
+      <p data-cy="noPeopleMessage">
+        There are no people on the server
+      </p>
+    );
+  }
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
The names of the columns can be written in an array and re-rendered through the `map`.